x/concentrated-liquidity/math/tick.go

```diff
@@ -45,6 +45,23 @@ func TickToSqrtPrice(tickIndex int64) (sdk.Dec, sdk.Dec, error) {
 	return price, sqrtPrice, nil
 }
 
+// TickToSqrtPriceBigDec returns the sqrtPrice given a tickIndex
+// as a BigDec.
+func TickToSqrtPriceBigDec(tickIndex int64) (osmomath.BigDec, error) {
+	price, err := TickToPrice(tickIndex)
+	if err != nil {
+		return osmomath.BigDec{}, err
+	}
+
+	// Determine the sqrtPrice from the price
+	// TODO: should we use MonotonicSqrtBigDec here?
+	sqrtPrice, err := osmomath.MonotonicSqrt(price)
```

Comment:
Any thoughts on reimplementing this for `BigDec` only to accommodate `math.CalculateSqrtPriceToTickBigDec`?

Reply:
We discussed this on Slack and agreed to not re-implement it. Leaving this open for now to rereview prior to merge